yarn/src/main/scala/org/apache/spark/deploy/yarn/ApplicationMaster.scala

```diff
@@ -483,14 +483,14 @@ private[spark] class ApplicationMaster(
         } catch {
           case e: InvocationTargetException =>
             e.getCause match {
-              case _: InterruptedException =>
+              case e: InterruptedException =>
```

Comment:
Nit: reusing the name `e` is ambiguous but also unnecessary; this can be `_` too? (Yes the second clause will generate a warning without `: Throwable`